render.yaml

```diff
@@ -107,6 +107,9 @@ services:
     startCommand: |
       cd server
       node scripts/availability_dump.js --write-to-s3 --clear-log
+    domains:
+      - getmyvax.org
+      - www.getmyvax.org
```

Comment:
I *think* this has to be here otherwise new deployments will blow domains we configure in the web console away. Not 100%, though. @alexallain, we could pilot this by configuring `univaf.usdigitalresponse.org` in the web console and seeing if it survives without explicit configuration in `render.yaml`. (You’d need to create the DNS records in USDR’s Cloudflare account.)

Reply:
Just ran this experiment with @alexallain, and we need to configure `render.yaml` first. (But it was very quick and worked great!)